sklearn/tree/_export.py

```diff
@@ -777,8 +782,10 @@ def export_graphviz(
 
     Parameters
     ----------
-    decision_tree : decision tree classifier
-        The decision tree to be exported to GraphViz.
+    decision_tree : object
```

Comment:
Do I need to change this docstring back to the original version?

Reply:
What was your thinking when changing the docstring?

My (before hearing your thoughts) opinion would be to stick with the original

Reply:
I would prefer the original one. Does `export_graphviz` support regression trees? If yes, then we might need to update it then. (I only use `plot_tree` that supports both type of instances).

Reply:
I was trying to make the docstring consistent with that of `export_text`, which says

https://github.com/scikit-learn/scikit-learn/blob/f097f436451d5dec258149ab64e3f0943042dd92/sklearn/tree/_export.py#L938-L957

I'm not sure if it supports other types of `decision_tree` objects, but `DecisionTreeClassifier` and `DecisionTreeRegressor` can work. As follows is an example of using `DecisionTreeRegressor`.

<img src="https://user-images.githubusercontent.com/108576690/229511177-dd717e90-ef24-4270-9ee4-85717e786aab.png" width="60%" />

Reply:
I have currently adapted @jeremiedbb's suggestions, that is, to support only `DecisionTreeClassifier` and `DecisionTreeRegressor`.